config/initializers/assets.rb

```diff
@@ -1 +1,3 @@
 Rails.application.config.assets.precompile += %w( print.css )
+Rails.application.config.assets.precompile += %w( I18n.css )
+Rails.application.config.assets.precompile += %w( sidebar.js )
```

Comment:
You can have all these in one line to avoid repetition...

Reply:
Ok I will do these